test/freenet/node/NewPacketFormatTest.java

```diff
@@ -33,7 +33,7 @@ public void setUp() {
 	@Test
 	public void testEmptyCreation() throws BlockedTooLongException {
 		NewPacketFormat npf = new NewPacketFormat(null, 0, 0);
-		PeerMessageQueue pmq = new PeerMessageQueue();
+		PeerMessageQueue pmq = new PeerMessageQueue(new MersenneTwister(new byte[16]));
```

Comment:
Apparently we don't care about the randomness in the tests, otherwise the tests would have failed.
If we don't care about it anyways, how about using `new DummyRandomSource(0)` rather than the impressive looking (but otherwise irrelevant) MersenneTwister?

Reply:
Yeah. And for extremely fast nodes, even this may slow down performance, could be replaced by an atomic counter 1-10.

Reply:
@torusrxxx I’m careful about deterministic values, because if different nodes operate in lockstep, that could cause unexpected behavior. It should not be problematic: according to https://github.com/js42721/fastrandom Mersenne Twister creates 1 billion random numbers in less than 10s, so this should not be a bottleneck for our use-case. When routing only SSKs on a 10gigabit/s  connection (1.25 gigabyte per second), we’ll have roughly 1 million packages per second, so we have about factor 100 buffer until Mersenne Twister could become a bottleneck.

@bertm switched to DummyRandomSource(1234) — 1234, because that value is often used in other places.